hack/ipu_host_deploy.sh

```diff
@@ -0,0 +1,26 @@
+cd cluster-deployment-automation
+source ocp-venv/bin/activate
+
+export PATH=$PATH:"/usr/local/go/bin"
+
+# We need to overwrite the buildID to ensure that Jenkin's ProcessTreeKiller does not clean up the containers spawned by CDA on job completion. https://wiki.jenkins.io/display/JENKINS/ProcessTreeKiller
+# Specifically, we want the local container registry to persist so we do not need to rebuild images each time we run a new job.
+export BUILD_ID=dontKillMe
```

Comment:
I don't think we need this now...this is a Jenkins specific dial to manage how Jenkins handles process started by jobs...if we are decoupled from Jenkins with this change, this is not required.